tiledb/api/src/filter/strategy.rs

```diff
@@ -83,53 +83,51 @@ fn prop_bitwidthreduction() -> impl Strategy<Value = FilterData> {
 fn prop_compression_delta_strategies(
     input_datatype: Option<Datatype>,
 ) -> Vec<BoxedStrategy<CompressionType>> {
-    if let Some(input_datatype) = input_datatype {
-        if input_datatype.is_real_type() {
-            let delta = any::<Datatype>()
-                .prop_filter(
-                    "input_datatype is floating-point, input must not be Any",
-                    move |dt| {
-                        if input_datatype.is_real_type() {
-                            !dt.is_real_type() && *dt != Datatype::Any
-                        } else {
-                            !dt.is_real_type()
-                        }
-                    },
-                )
-                .prop_map(|dt| CompressionType::Delta {
-                    reinterpret_datatype: Some(dt),
-                });
-
-            let double_delta = any::<Datatype>()
-                .prop_filter(
-                    "input_datatype is floating-point, input must not be Any",
-                    move |dt| {
-                        if input_datatype.is_real_type() {
-                            !dt.is_real_type() && *dt != Datatype::Any
-                        } else {
-                            !dt.is_real_type()
-                        }
-                    },
-                )
-                .prop_map(|dt| CompressionType::DoubleDelta {
-                    reinterpret_datatype: Some(dt),
-                });
-
-            return vec![delta.boxed(), double_delta.boxed()];
+    fn validate_type(
+        in_dtype: &Option<Datatype>,
+        out_dtype: &Datatype,
```

Comment:
I stared at this a lot longer than I probably would have if it were named `reinterpret_datatype`.
The reinterpret datatype isn't the same as the output type, the output type will soon be blob (https://github.com/TileDB-Inc/TileDB/pull/4988/files#diff-f46f30278b5b0f9992c18dab02d507dc5f489bc304d82a6404c692da761ddaf3)